pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -325,6 +325,7 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(
 				err)
 			dswp.desiredStateOfWorld.AddErrorToPod(uniquePodName, err.Error())
 			allVolumesAdded = false
+			continue
```

Comment:
To ensure I'm reading this correctly... in the previous code, we would print both the `Failed to add volume...` log message and the `Added volume...` log message. You are correct, that seems very confusing.

Reply:
That was the case without my fix.